### ISSUE TITLE
require-indexer-name, require-parameter-name

### ISSUE THREAD

**Issue by goodmind** [gajus/eslint-plugin-flowtype#424]
Basically these two rules would disallow:

```js
type F = (A, B) => C // error
type F = (a: A, b: B) => C // no error
```

```js
type A = { [string]: number } // error
type B = { [key: string]: number } // no error
```

I tried to add these, but tests doesn't seem to work

**Comment by gajus** [comment]
> I tried to add these, but tests doesn't seem to work

Can you clarify what do you mean?

**Comment by goodmind** [comment]
@gajus `ObjectTypeIndexer` doesn't seem to have `id` property like in Flow AST.
![image](https://user-images.githubusercontent.com/3275424/62778629-4f69ba80-baca-11e9-9329-38909b33398f.png)

**Comment by goodmind** [comment]
When I tried it in astexplorer, there was `id` property

**Comment by goodmind** [comment]
https://github.com/babel/babel-eslint/blob/master/lib/babylon-to-espree/convertAST.js#L76 huh

**Comment by gajus** [comment]
That particular issue seems to be with babel-eslint rather than eslint-plugin-flowtype.

**Comment by goodmind** [comment]
@gajus I found you have workaround in `utils/getParameterName.js`
